survey/5-tech.yml

```diff
@@ -332,4 +336,16 @@ questions:
       - Neo4j
       - Apache Cassandra
       - BigTable
+      - SQLite
```

Comment:
Sqlite is not really used in the backend and more a client database. I would rather remove it from here and update the question to make it more explicit. Like where do you store your large data in the backend or something

Reply:
Apparently, that's no longer the case. SQLite can now be used in production after the introduction  of [libSQL](https://turso.tech/libsql), and it's been receiving a lot of hype recently. Also Cloudflare recently introduced something similar on their developer platform.
For more clarification i will add LibSQL ?

Reply:
It is still a niche, cloudflare uses it for a specific uses case (for their workers serverless platform) because of it speed and simplicity. I dont think enterprises really use it widely

Reply:
Yeah, I agree it's not used that much on enterprise levels. , but it can be used for side projects. At least I am using it for some projects right now. so i will choose it 😀.  It's good to have more options anyway.